.github/workflows/macos_build.yaml

```
@@ -1,87 +1,280 @@
+          echo "Signing main application bundle: ${APP_BUNDLE_PATH}"
+          codesign --force --deep --strict --options=runtime --timestamp \
+            --entitlements "${ENTITLEMENTS_PATH}" \
+            --sign "${SIGNING_IDENTITY}" \
+            "${APP_BUNDLE_PATH}"
+        shell: bash
+
+      - name: 🔎 Verify Signatures After Signing
+        shell: bash
+        run: |
+          APP_BUNDLE_PATH="apps/flites/build/macos/Build/Products/Release/${{ env.APP_NAME_LOWER }}.app"
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)